libqtile/state.py

```diff
@@ -31,10 +32,17 @@ def __init__(self, qtile):
         # configurations.
         self.groups = {}
         self.screens = {}
+        self.focusHistory = {}
```

Comment:
New code should use `snake_case` rather than `mixedCase`: http://docs.qtile.org/en/latest/manual/hacking.html#coding-style